BackwardElimination.py

```diff
@@ -0,0 +1,47 @@
+import pandas as pd
+import statsmodels.api as sm
+import numpy as np
+
+
+def backward_elimination(x, y, sl):
+    x = pd.DataFrame(np.append(np.ones((x.shape[0], 1)).astype(int), x, axis=1))
+    rSqData = pd.DataFrame(r_sq_elimination(x, y))
+    finalModel = p_elimination(rSqData, y, sl)
+    return finalModel
+
+
+def r_sq_elimination(x, y):
+    x = pd.DataFrame(x)
+    ols_regeressor = sm.OLS(exog=x, endog=y).fit()
+    bestValue = float("{0:.4f}".format(ols_regeressor.rsquared_adj))
+    noOfColumn = x.shape[1]
+    bestModel = pd.DataFrame(x)
+    foundNew = False
+    for i in range(1, noOfColumn):
+        temp = x.drop(x.columns[i], axis=1)
+        ols_regeressor = sm.OLS(endog=y, exog=temp).fit()
+        rValue = float("{0:.4f}".format(ols_regeressor.rsquared_adj))
+        if bestValue < rValue:
+            bestValue = rValue
+            bestModel = temp
+            foundNew = True
+
+    if foundNew == True:
+        bestModel = r_sq_elimination(bestModel, y)
+        return bestModel
+    else:
+        return bestModel
```

Comment:
In any case, `bestModel` needs to returned, could this be used without needing `else` check?

```
if foundNew == True:
        bestModel = r_sq_elimination(bestModel, y)

return bestModel
```